utils/zerovec/src/map/map.rs

```diff
@@ -377,8 +377,8 @@ where
 impl<'a, K, V> ZeroMap<'a, K, V>
 where
     K: ZeroMapKV<'a> + ?Sized + Ord,
-    V: ZeroMapKV<'a, Container = ZeroVec<'a, V>> + ?Sized,
-    V: AsULE + Copy,
+    V: ZeroMapKV<'a> + ?Sized,
+    V: Copy,
```

Comment:
@Manishearth I want to draw your attention to this: I used `V::Container::zvl_get_as_t` to make it such that `V: Copy` is the only bound we need any more for `ZeroMap::get_copied` and friends. I'm not sure if it has a code size impact, but if everything is inlined, it should be okay.